apluslms-yamlidator/apluslms_yamlidator/utils/collections.py

```diff
@@ -93,44 +93,54 @@ def data_updated(self, key, data):
     def get_data(self):
         return self._data
 
+    def get_root(self):
+        return self._root
+
+    @property
+    def _root(self):
```

Comment:
I'm not sure if `_root` is usefull here, as `get_root` is the only one used. Alternatively, last line of this property should probably have line `parent._root` and not `parent.get_root()`, Currently, the flow of the code jumps a bit around.